actors/common/src/main/java/org/sunbird/learner/util/CourseBatchUtil.java

```diff
@@ -28,4 +33,40 @@ public static void syncCourseBatchForeground(String uniqueId, Map<String, Object
             + esResponse,
         LoggerEnum.INFO.name());
   }
+
+  public static String doOperationInEkStepCourse(
```

Comment:
CourseBatchSchedulerUtil.doOperationInEkStepCourse -> how is this function different then mentioned here?